assets/section-email-signup-banner.css

```diff
@@ -0,0 +1,35 @@
+.email-signup-banner__box .email-signup-banner__heading {
+  margin-bottom: 0;
+}
+
+.email-signup-banner__box > * + .newsletter__subheading {
+  margin-top: 2rem;
+}
+
+.email-signup-banner__box .newsletter__subheading p {
+  margin: 0;
+}
+
+.email-signup-banner-background {
+  width: 100%;
+  height: 100%;
+  position: relative;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+@media only screen and (min-width: 750px) {
+  .email-signup-banner-background-mobile {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 749px) {
+  .email-signup-banner-background:not(.email-signup-banner-background-mobile) {
+    display: none;
+  }
+}
+
+.email-signup-banner .banner__media {
+  overflow: hidden;
+}
```

Comment:
A couple things I'm wondering about the styling of the section: 

- When the section is set to large and is the only section, should we add a `height: 100%` to prevent [this white space](https://screenshot.click/21-10-3f45t-3xt7x.png) from showing? 
- When the section isn't using an image and is the only section, should we center it vertically by making the parent height `100%` ? [screenshot](https://screenshot.click/21-10-t7qf7-gg6t8.png)